src/Resources/contao/classes/BackendUser.php

```diff
@@ -111,6 +123,32 @@ protected function __construct()
 		$this->strHash = \Input::cookie($this->strCookie);
 	}
 
+	public static function getInstance()
+	{
+		/** @var TokenInterface $token */
```

Comment:
Shouldn't there be a `if (static::$objInstance !== null)` check somewhere? This code will be run upon every call instead of just upon the first call. Intended?

Reply:
Good point, should be checked then.